completed/administration.js

```diff
@@ -1,15 +1,16 @@
 import axios from "axios";
 import { getAccessToken } from "./auth.js";
 
+const authorizationConfiguration = {
+  clientId: process.env.ADMINISTRATION_CLIENT_ID,
+  clientSecret: process.env.ADMINISTRATION_CLIENT_SECRET,
+  audience: process.env.ADMINISTRATION_AUDIENCE
+};
```

Comment:
I like the idea of assigning this config object once per API client, rather than every single time. Open to feedback on it though.